aws/ebs_test.go

```diff
@@ -85,6 +86,24 @@ func findEBSVolumesByNameTag(t *testing.T, session *session.Session, name string
 	return volumeIds
 }
 
+func findallEBSVolumesByStatus(t *testing.T, session *session.Session, status string) ([]*string, error) {
+	statusFilter := ec2.Filter{Name: aws.String("status"), Values: []*string{aws.String(status)}}
+
+	output, err := ec2.New(session).DescribeVolumes(&ec2.DescribeVolumesInput{
+		Filters: []*ec2.Filter{&statusFilter},
+	})
+	if err != nil {
+		return []*string{}, err
```

Comment:
I was wondering if returning nil will not be a better representation of an empty slice?